Kconfig

```diff
@@ -447,6 +447,22 @@ config DFU_PID
 	hex "PID to use for DFU image"
 
 endif # DFU_BINARY
+
+config OUTPUT_STRIP_PATHS
+	bool "Strip absolute paths from binaries"
+	default y
+	depends on ARCH_TOOLCHAIN_GNU
+	---help---
+	  Cmake build provide absolute paths to compile files. If __FILE__
```

Comment:
I think other places use TABs for initial help indentation. @anchao could you please update in next PR?

Reply:
@pkarashchenko Thanks for review, I will submit a new PR to fix this.

Reply:
FIX:
https://github.com/apache/nuttx/pull/12835